tests/playback.js

```diff
@@ -117,6 +129,16 @@ queue_next_packet = function () {
 var bytes_processed = 0;
 
 do_packet = function () {
+    // Avoid having an excessive queue buildup
+    if (rfb._flushing) {
```

Comment:
The performance test is not doing normal stuff. It is flooding the code with updates without any form of flow control. :)
This chunk is to have some semblance of rate control to get it a bit more closer to the behaviour a normal connection would see. Normally this code would be in the VNC server, so it makes sense that we put it in the performance test (which has to emulate the server after all).